src/common/xo/index.js

```diff
@@ -538,6 +538,33 @@ export const ejectCd = vm => (
   xo.call('vm.ejectCd', { id: resolveId(vm) })
 )
 
+export const setVmBootOrder = (vm, order) => (
+  xo.call('vm.setBootOrder', {
+    id: resolveId(vm),
+    order
+  })
+)
+
+export const addVdi = (vm, vdi, position, mode, bootable) => (
+  xo.call('vm.attachDisk', {
```

Comment:
Disk section, I don't know. It calls the VM API, it's ok it  wouldn't create a big confusion in the code, but ...